its/ruling/src/test/resources/sonar-server/java-S5863.json

```diff
@@ -0,0 +1,8 @@
+{
+'org.sonarsource.sonarqube:sonar-server:src/test/java/org/sonar/server/computation/task/projectanalysis/api/measurecomputer/MeasureComputerDefinitionImplTest.java':[
+156,
+],
+'org.sonarsource.sonarqube:sonar-server:src/test/java/org/sonar/server/user/DefaultUserTest.java':[
+33,
```

Comment:
I'm still worried about the noise we will generate when reporting similar FPs.
The fact that we will consider this obvious FP as a "major bug" doesn't help...

Maybe we should ask for a third mind?

Reply:
After discussions, we decided to change the strategy. Instead of having an exception using packages name, we will try to base the exception on unit test method name containing some keywords (e.g. "equals", "hash_code")